presto-main/src/test/java/com/facebook/presto/type/TestCharacterStringCasts.java

```diff
@@ -61,9 +66,103 @@ public void testCharToVarcharCast()
     @Test
     public void testVarcharToCharSaturatedFloorCast()
     {
-        assertEquals(varcharToCharSaturatedFloorCast(10L, utf8Slice("1234567890")), utf8Slice("1234567890"));
-        assertEquals(varcharToCharSaturatedFloorCast(10L, utf8Slice("123456789")), utf8Slice("12345678"));
-        assertEquals(varcharToCharSaturatedFloorCast(10L, utf8Slice("12345678901")), utf8Slice("1234567890"));
-        assertEquals(varcharToCharSaturatedFloorCast(10L, utf8Slice("")), utf8Slice(""));
+        // Encoded in UTF-8
+        byte[] nonBmpCharacter = codePointToUtf8(0x1F50D).getBytes();
```

Comment:
There are two options to avoid byte[] arrays:
- `"\uD83D\uDD0D"` (then `.getBytes(UTF-8)` will encode this as one, four byte character)
- `new String(Character.toChars(0x1F50D))`

i went for the second one. Indeed looks better than low-level byte handling and now I checked this is equivalent (test updated and still passes :) )